### PR TITLE
[SAMPLE] BigQuery Dialect setup

### PR DESCRIPTION
**Pull request by vijaypavann-db**
DO NOT MERGE THIS PR

### CODE REVIEW

File: src/databricks/labs/remorph/bigquery/bigquery.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import logging
+from typing import ClassVar
+
+from sqlglot.dialects.bigquery import BigQuery
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=function-redefined
+class BigQuery(BigQuery):
```
Comment (sundarshankar89) [inline src/databricks/labs/remorph/bigquery/bigquery.py:10]: @vijaypavann-db `config.py` also needs modification to use this class

**Review by sundarshankar89 — CHANGES_REQUESTED** [review]

File: src/databricks/labs/remorph/bigquery/bigquery.py (hunk 1 above)
Comment (vijaypavann-db) [reply src/databricks/labs/remorph/bigquery/bigquery.py:10]: Done, Updated `config.py`

Comment (nfx) [inline src/databricks/labs/remorph/bigquery/bigquery.py:10]:
```suggestion
from sqlglot.dialects import bigquery as orig

logger = logging.getLogger(__name__)


class BigQuery(orig.BigQuery):
```

Comment (nfx) [reply src/databricks/labs/remorph/bigquery/bigquery.py:10]: don't disable linters

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/remorph/bigquery/bigquery.py (hunk 1 above)
Comment (vijaypavann-db) [reply src/databricks/labs/remorph/bigquery/bigquery.py:10]: Done

**Pull request closed without merging**